nejma/layers.py

```diff
@@ -81,4 +86,57 @@ def clean_expired(self):
                     del self.groups[group][channel]
 
 
+class RedisLayer(ChannelLayer):
+    def __init__(self, redis_host="redis://localhost", prefix="group"):
+        self.initialized = False
+        self.send = None
+        self.redis_host = redis_host
+        self.prefix = prefix
+
+    ### utility functions ###
+
+    def group_prefix(self, group):
+        return f"{self.prefix}_{group}" if not group.startswith(
+            f"{self.prefix}_") else f"{group}"
+
+    async def _initialize(self):
+        self.redis: aioredis.Redis = await aioredis.create_redis(self.redis_host)
```

Comment:
Thanks for this great implementation @MrNaif2018. Would there be any advantage in using
`aioredis.create_redis_pool(self.redis_host)` instead of `aioredis.create_redis(self.redis_host)`. According to aioredis docs this should give 

> Redis client bound to pool of connections (allows auto-reconnecting).

Reply:
Yes, auto-reconnecting and in general connection pool would be better. Will change that in next commit.